VirtoCommerce.Storefront/Domain/Cart/CartService.cs

```diff
@@ -130,6 +130,31 @@ public virtual async Task<IPagedList<ShoppingCart>> SearchCartsAsync(CartSearchC
             });
         }
 
+        public virtual async Task<IPagedList<CartDescriptionDto>> SearchAccountListsAsync(CartSearchCriteria criteria)
```

Comment:
We should use and change the existing `SearchCartsAsync` method, it should return response with multiple carts and totalCount.
PLease ensure proper CartType is used - in APIListController - wishlist, and default in ICartBuilder